release_notes/ocp-4-9-release-notes.adoc

```diff
@@ -383,6 +383,22 @@ Starting in {product-title} {product-version}, Operator Lifecycle Manager (OLM)
 Kubernetes 1.22 introduces link:https://kubernetes.io/docs/reference/using-api/deprecation-guide/#customresourcedefinition-v122[several notable changes] to `v1` of the `CustomResorceDefinition` API.
 ====
 
+[id="ocp-4-9-olm-admin-error-reporting"]
+==== Enhanced error reporting for administrators
+Because administrators should not require understanding of the interaction process between the various low-level APIs or access to the OLM pod logs to successfully debug such issues, {product-title} {product-version} introduces the following enhancements in OLM to provide administrators with more comprehensible error reporting and messages:
+
+[id="ocp-4-9-olm-operator-group-status-conditions"]
+Updating Operator group status conditions::
+Previously, if a namespace contained multiple Operator groups or could not find a service account, the status of the Operator group would not report an error. With this enhancement, the error will update the status condition of the Operator group.
+
+[id="ocp-4-9-olm-subscription-conditions"]
+Indicating the reason for install plan failures::
+Before this release, if an install plan failed, the subscription condition would not state why the failure occurred. Now, if an install plan fails, the subscription status condition indicates why.
+
+[id="ocp-4-9-olm-subscription-resolution-errors"]
+Indicating resoltuion conflicts on subscription statuses::
+Since resolution treats all components in a namespace as a single unit, if a resolution failure occurs, all subscriptions on the namespace indicate the error.
```

Comment:
s/Since/Because/

If technically accurate:

s/resolution treats/dependency resolution treats/

(keeping the 2nd resolution as-is would be fine in this case)

s/indicate/now indicate/